package-lock.json

```diff
@@ -2029,7 +2029,7 @@
     },
     "dotenv": {
       "version": "5.0.1",
-      "resolved": "https://registry.npmjs.org/dotenv/-/dotenv-5.0.1.tgz",
+      "resolved": "http://registry.npmjs.org/dotenv/-/dotenv-5.0.1.tgz",
```

Comment:
a little bit weird, why become http?

Reply:
Not sure what's wrong with npm, i will take a look